recipes/homopolish/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "homopolish" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/chengjun109/{{name}}/releases/download/1/{{name}}_{{version}}.tar.gz
+  sha256: 097cb49f098e9aafdf77029368c007c93485b60fa5f28ca5b469ad23b0581189
+build:
+  entry_points:
+    - homopolish = homopolish.homopolish:main
+  number: '0'
+  noarch: python
+  script: python -m pip install . -vv
+requirements:
+    host:
+         - pip
+         - numpy=1.18.5
+    run:
+         - mash=2.1.1
+         - scikit-learn=0.21.3
+         - minimap2=2.17
```

Comment:
Does it really require these exact versions or are later versions also acceptable? I assume all of the `=` can be replaced with `>=`.

Reply:
Yes, We are worried  about the conflict of the environment.
Additionally, some parameters are not applicable in new version.